source/MoorDyn2.cpp

```diff
@@ -108,11 +108,11 @@ moordyn::MoorDyn::MoorDyn(const char* infilename, int log_level)
 		_basepath = _filepath.substr(0, lastSlash + 1);
 	}
 
-	LOGMSG << "\n Running MoorDyn (v2.0.0, 2023-09-18)" << endl
+	LOGMSG << "\n Running MoorDyn (v2.3.1, 2024-5-24)" << endl // TODO: can we automate this?
```

Comment:
The version for sure! The date would be quite annoying... However I think we can just drop it, since we have now automated releases with the timestamp. Everyone can check when do we release a new version

Reply:
Agreed, as long as the version is up to date users can reference the release.